src/users/containers/users/users.component.ts

```diff
@@ -8,6 +8,7 @@ import {UserListApiModel} from '../../models/userform.model';
 @Component({
   selector: 'app-prd-users-component',
   templateUrl: './users.component.html',
+  styleUrls: ['./users.component.scss']
```

Comment:
plz remove the css... leave it as it is...

Reply:
Hi Praveen, i have reviewed the original pr and i have found inconsistency in that we should be adopting a common styles across our manage org\approve app and not creating a custom styles.  No criticism of your work however it was my bad for not asking you the context as to why you created a new css to compensate for inconsistancy.

I have seen a comment which hasnt been responded to on that note specifically.  Can i request you update the PR based on my feedback?